updater/templates/updater_screen_four.html

```diff
@@ -0,0 +1,70 @@
+<div class="text-center col-xs-2"><br/>
```

Comment:
Not sure how to solve the dilemma of good code/view separation and the requirement to have translations.   Need to think about this issue.   I think the .HTML file need to be PHP with translate but no business logic...

Reply:
we can assign translate to it, i will modify the html files to get the translate feature and commit soon

Reply:
Curious, how this translation takes place, changed everything to xlt() still no translation takes place on screen
![screenshot from 2018-07-03 13-17-39](https://user-images.githubusercontent.com/18109258/42205958-7de3a7ac-7ec3-11e8-8d01-41763e2a7e22.png)
brought the language to bengali and took the screenshot

Reply:
You have to make language translation table additions to the database.
Previously, these where compiled from working documents.  None of that has taken place in a long time!  বাংলার জন্য কোনও অনুবাদ হতে পারে না!

Reply:
Need Bangla translations!

Reply:
Does that mean if i use a word, i need to add the values to the language table in various languages?

Reply:
Yes.  Simple translation matrix.
If you have a language set and translations on, then the xlt() function fires, queries the database for the replacement text, and puts that out.
Naturally, the base inclination is to provide an API hook to a text translation tool.  That is not such a good idea in technical settings.  It is a very bad one in medical.  Most medical terms will not be in your typical translation program (even Nuance/Dragon/inuit has a special medical dictionary for speech dictation) but also euphemisms kind of get mashed up.  You know this, I am quite sure.  Thus a local function is used, and local terminology.  That is great, but of course it is not maintained, and thus is not useful.  It is no less useless than inaccurate translations though.  Broken is broken.

Reply:
so at some point it may revive, so its useful to keep the strings in translation right?

Reply:
Yes, absolutely...
and I strongly encourage you to provide the Bangla translations for setup...and insist that we help get at least the Spanish translation for the feature done too.

Reply:
Bengali is not my native language, i can provide translations by using google translate.

Reply:
Uh....I guess I just assumed it was as you went looking at it.  Fluent language translations only.  If you are technically (not medically) fluent in at least one language other than English, you could provide translations for that so as to ensure that the translation table items for the updater are in place.